x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -695,17 +680,31 @@ export const RulesList: React.FunctionComponent = () => {
       {
         field: 'executionStatus.status',
         name: i18n.translate(
-          'xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.statusTitle',
-          { defaultMessage: 'Status' }
+          'xpack.triggersActionsUI.sections.rulesList.rulesListTable.columns.lastResponseTitle',
```

Comment:
@Zacqary Make sure to update also the current `status filter` to be called Last response

Reply:
A new `Status` filter needs to be created as well, but I would start with renaming current one to last response.